src/libs/Share/navigateToShare.js

```diff
@@ -0,0 +1,15 @@
+import isArray from 'lodash/isArray';
+
+import ROUTES from '../../ROUTES';
+import Navigation from '../Navigation/Navigation';
+
+const navigateToShare = (share) => {
+    if (!share || !share.data) return;
+    if (isArray(share.data) && share.data.length === 0) return;
```

Comment:
note, for the final code it might be nice to have a `shareDataExists` utility fn